test/allowlist_for_publicAPI.json

```diff
@@ -3250,274 +3250,15 @@
     "runtime_validation",
     "runtime_validation_disabled"
   ],
-  "torch.utils.data.communication.eventloop": [
-    "IterDataPipe",
-    "MapDataPipe"
-  ],
-  "torch.utils.data.communication.iter": [
-    "IterDataPipe"
-  ],
-  "torch.utils.data.communication.map": [
-    "MapDataPipe"
-  ],
   "torch.utils.data.dataloader": [
-    "Any",
-    "BatchSampler",
-    "Callable",
-    "Dataset",
-    "ExceptionWrapper",
-    "Generic",
-    "IterDataPipe",
-    "Iterable",
-    "IterableDataset",
-    "List",
-    "Optional",
-    "RandomSampler",
-    "Sampler",
-    "Sequence",
-    "SequentialSampler",
-    "TypeVar",
-    "Union",
+    "_DatasetKind",
```

Comment:
Ditto

Reply:
Sorry to be more clear, this should be removed according to @albanD and unless you remove the other one as well, the CI will fail.